tests/phpunit/elementor/test-widgets.php

```diff
@@ -67,7 +69,7 @@ public function test_controlsDefaultData() {
 
 				$error_msg = sprintf( 'Widget: %1$s, Control: %2$s', $widget->get_name(), $control['name'] );
 
-				if ( empty( $control['default'] ) ) {
+				if ( Utils::is_empty( $control['default'] ) ) {
```

Comment:
Please remove the file from PR.

Reply:
If I remove this file from the PR, the PR will fail the PHPUnit integration tests

Reply:
Which change cause it to fail?